release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -196,6 +196,27 @@ By default, the following network connections are checked:
 ** the internal API load balancer
 ** the external API load balancer
 
+[id="ocp-4-6-storage"]
+=== Storage
+
+[id="ocp-4-6-csi-driver-cso"]
+==== CSI drivers now managed by the Cluster Storage Operator
+The Container Storage Interface (CSI) drivers for AWS Elastic Block Store (EBS), Red Hat Virtualization (oVirt), and OpenStack Manila shared file system service are now managed by the Cluster Storage Operator in the {product-title} back end.
+
+For AWS EBS and oVirt, this feature installs the CSI Driver Operator and driver in the `openshift-cluster-csi-drivers` namespace by default. For Manila, the CSI Operator and driver are installed in the `openshift-manila-csi-driver` namespace.
```

Comment:
@bobfuru Sorry for I did not say this clearly. Manila CSI Operator installed in `openshift-cluster-csi-drivers` namespace, Manila CSI Drivers installed in `openshift-manila-csi-driver` namespace.

$ oc get deploy -n openshift-cluster-csi-drivers 
NAME                         READY   UP-TO-DATE   AVAILABLE   AGE
manila-csi-driver-operator   1/1     1            1           8h

$ oc get deploy -n openshift-manila-csi-driver
NAME                                    READY   UP-TO-DATE   AVAILABLE   AGE
openstack-manila-csi-controllerplugin   1/1     1            1           8h

$ oc get ds -n openshift-manila-csi-driver
NAME                              DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
csi-nodeplugin-nfsplugin          6         6         6       6            6           <none>          8h
openstack-manila-csi-nodeplugin   6         6         6       6            6           <none>          8h

Reply:
@qinpingli Good to know, thanks. It's important we get it cleared up as that's easy to see where it could be confusing. I've updated the info in the release note, so please let me know if it looks good now.